agent/util-scripts/samples/pbench-list-tools/test-12.03

```diff
@@ -0,0 +1 @@
+test-12.02/
```

Comment:
Why is there a trailing slash here when there isn't one in `agent/util-scripts/samples/pbench-list-tools/test-12.04`?

Reply:
I guess because it's a link to the 12.02 directory (they share the same setup).

Reply:
Yes, but the same is true of '`12.04`, and that one lacks the slash.

Reply:
I think I was inconsistent when creating the links:

```
ls -l
drwxrwxr-x. 3 nick nick 20 May 11 22:07 test-12.02
lrwxrwxrwx. 1 nick nick 11 May 11 22:11 test-12.03 -> test-12.02/
lrwxrwxrwx. 1 nick nick 10 May 11 22:23 test-12.04 -> test-12.02
```